.github/ISSUE_TEMPLATE/icon_request.yml

```diff
@@ -0,0 +1,73 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post. Also take a look at our
+        [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md)
+        for more details.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+      description: >-
+        For non-web brands you can add a relevant link or simply put "None" if you don't
+        think there's a website.
+
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Alexa Rank
+      description: >-
+        Provide either an [Alexa rank](https://www.alexa.com/siteinfo) or the number
+        of GitHub stars. If these are not available or insufficient you can still open
+        the issue. Please provide any information regarding the brand's popularity you
+        think is relevant in the description below.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. We strongly encourage you to do
+        some research here as it will help implement your request faster, although you
+        can set this to none if you are unable to find any resources.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional Comments
+      description: |
+        Is there anything else we should know about the brand? For example:
+        - Are there multiple options for the logo and/or color?
+        - Is the icon licensed?
+        - If the brand isn't popular enough by the default criteria, how can we assess the popularity instead?
+        - If you think the brand might not be accepted for other reasons, why do you think it should be considered?
+
+  - type: dropdown
+    attributes:
+      label: Contributing
+      description: >-
+        This is an open source project and we welcome contributions. Do you want help
+        implement this?
+      options:
+        - "Yes"
+        - "No"
```

Comment:
Not sure about the wording on this. Also not sure if this should be a required field. I'm leaning towards yes just to draw more attention to it.

Reply:
> Not sure about the wording on this.

How about: "This is an open source project and we welcome contributions. Do you want to work on this issue?" - mainly looking to simplify the language by omitting "implement"

> Also not sure if this should be a required field. I'm leaning towards yes just to draw more attention to it.

I think it's okay for it to be a required field, selecting "No" isn't that difficult right :thinking:

Reply:
Not sure what happened I don't think I got the notifications for these comments? New description sounds better though, so I'll change those.

> I think it's okay for it to be a required field, selecting "No" isn't that difficult right thinking

:+1: